app/components/post_textbox/post_textbox.js

```diff
@@ -96,6 +102,66 @@ export default class PostTextbox extends PureComponent {
         this.autocomplete = c;
     };
 
+    attachFileFromCamera = async () => {
+        try {
+            this.props.actions.closeModal();
+            const image = await ImagePicker.openCamera({
```

Comment:
Can't select multiple from the camera, but we can perform cropping if that is needed 😄